src/core/evaluator.js

```diff
@@ -802,7 +802,7 @@ var PartialEvaluator = (function PartialEvaluatorClosure() {
               // Error in the font data -- sending unsupported feature
               // notification.
               this.handler.send("UnsupportedFeature", {
-                featureId: UNSUPPORTED_FEATURES.font,
+                featureId: UNSUPPORTED_FEATURES.errorType3FontLoad,
```

Comment:
Since all the other font-related ones follow a `errorFont...` pattern, how about naming this one `errorFontLoadType3` instead since that's also consistent with the actual method name used above?